esy-core/esy-build-package/NodeResolution.re

```diff
@@ -2,16 +2,18 @@ module PackageJson = {
   type t = {
     name: string,
     main: option(string),
-    browser: option(string)
+    browser: option(string),
   };
   let of_json = data =>
     Yojson.Safe.Util.(
-      try {
-        let name = member("name", data) |> to_string;
-        let main = member("main", data) |> to_string_option;
-        let browser = member("browser", data) |> to_string_option;
-        Result.Ok({name, main, browser});
-      } {
+      try (
+        {
+          let name = member("name", data) |> to_string;
+          let main = member("main", data) |> to_string_option;
+          let browser = member("browser", data) |> to_string_option;
+          Result.Ok({name, main, browser});
+        }
+      ) {
```

Comment:
hm... seems like a regression in readability.

Reply:
Yes, but we had to wrap all of the try/switch arguments in parens to make room for `try async` etc eventually. I think we just need to tweak the formatter so that it at least hugs the braces like `({`